prisma/schema.prisma

```diff
@@ -62,6 +64,57 @@ model NoteImage {
   @@index([noteId])
 }
 
+model Article {
+  id      String @id @default(cuid())
+  title   String
+  content String
+  isPublished Boolean @default(false)
+  
+  publishedAt DateTime?
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  owner   User   @relation(fields: [ownerId], references: [id], onDelete: Cascade, onUpdate: Cascade)
+  ownerId String
+  
+  category    ArticleCategory? @relation(fields: [categoryId], references: [id], onDelete: Cascade, onUpdate: Cascade)
+  categoryId  String?
+
+  images ArticleImage[]
+
+  // non-unique foreign key
+  @@index([ownerId])
+  @@index([categoryId])
+  // This helps our order by in the user search a LOT
+  @@index([ownerId, updatedAt])
+}
+
+model ArticleImage {
+  id          String  @id @default(cuid())
+  altText     String?
+  contentType String
+  blob        Bytes
+
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  article     Article   @relation(fields: [articleId], references: [id], onDelete: Cascade, onUpdate: Cascade)
+  articleId   String
+
+  // non-unique foreign key
+    @@index([articleId])
+}
+
+model ArticleCategory {
+  id          String @id @default(cuid())
+  name        String
+  slug        String @unique
+  createdAt   DateTime @default(now())
+  updatedAt   DateTime @updatedAt
+
+  articles    Article[]
+}
```

Comment:
Well done, Alex. This was a tricky update. You now need to [load and display article data](https://gedsargent.github.io/code-monster/guides/epic-news/displaying-articles/load-and-display-article-data/) on the news route.